src/parser.rs

```diff
@@ -836,17 +836,20 @@ impl DyneinParser {
         }
     }
 
-    pub fn parse_put_content(
+    pub fn parse_dynein_format(
         &self,
-        initial_item: HashMap<String, AttributeValue>,
+        initial_item: Option<HashMap<String, AttributeValue>>,
         exp: &str,
     ) -> Result<HashMap<String, AttributeValue>, ParseError> {
         let result = GeneratedParser::parse(Rule::map_literal, exp);
         match result {
             Ok(mut pair) => {
                 let item = parse_literal(pair.next().unwrap())?.convert_attribute_value();
-                // put content must be map literal
-                let mut image = initial_item;
+
```

Comment:
I deleted the comment because `parse_dynein_format` is no longer used only for put. However I should have just updated the comment. So, I will restore the modified version.